symbols/arm9.yml

```diff
@@ -2202,6 +2295,22 @@ arm9:
         r0: monster ID
         r1: ?
         return: bool
+    - name: GetPlayerData
```

Comment:
Is this strictly the main character, or whoever happens to be leader? If the former, I think we usually use the term "hero" rather than "player".

Reply:
I was debating between "hero", "player" and "protagonist".  I settled on "player" because that's the one SkyTemple uses, but I'll change it since it seems like this repo already uses "hero".